iGCS/UserInterface/en.lproj/MainStoryboard.storyboard

```diff
@@ -1310,7 +1310,7 @@
         <scene sceneID="cVi-Id-eoD">
             <objects>
                 <tableViewController id="g0h-Hk-vw4" customClass="MissionItemTableViewController" sceneMemberID="viewController">
-                    <tableView key="view" clipsSubviews="YES" contentMode="scaleToFill" alwaysBounceVertical="YES" dataMode="prototypes" style="plain" separatorStyle="default" rowHeight="44" sectionHeaderHeight="22" sectionFooterHeight="22" id="J2H-Pu-dEF">
+                    <tableView key="view" clipsSubviews="YES" contentMode="scaleToFill" bounces="NO" alwaysBounceVertical="YES" dataMode="prototypes" style="plain" separatorStyle="default" rowHeight="44" sectionHeaderHeight="22" sectionFooterHeight="22" id="J2H-Pu-dEF">
```

Comment:
+1 looks good.  To be clear the long term functionality that is desired is
bounce with the header (Seq#, Command, Latitude...) staying still.  The
header separating from the map view looks like a software glitch.

On Sat, Jan 24, 2015 at 6:33 AM, David Leskowicz <notifications@github.com>
wrote:

> In iGCS/UserInterface/en.lproj/MainStoryboard.storyboard
> <https://github.com/fightingwalrus/iGCS/pull/159#discussion_r23496653>:
>
> > @@ -1310,7 +1310,7 @@
> >          <scene sceneID="cVi-Id-eoD">
> >              <objects>
> >                  <tableViewController id="g0h-Hk-vw4" customClass="MissionItemTableViewController" sceneMemberID="viewController">
> > -                    <tableView key="view" clipsSubviews="YES" contentMode="scaleToFill" alwaysBounceVertical="YES" dataMode="prototypes" style="plain" separatorStyle="default" rowHeight="44" sectionHeaderHeight="22" sectionFooterHeight="22" id="J2H-Pu-dEF">
> > +                    <tableView key="view" clipsSubviews="YES" contentMode="scaleToFill" bounces="NO" alwaysBounceVertical="YES" dataMode="prototypes" style="plain" separatorStyle="default" rowHeight="44" sectionHeaderHeight="22" sectionFooterHeight="22" id="J2H-Pu-dEF">
>
> Hmm, I've used it so long with the bounce that it feels unnatural to not
> have it. I could go either way on this one though.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fightingwalrus/iGCS/pull/159/files#r23496653>.
>